app/style/ImportScriptModal.less

```diff
@@ -4,8 +4,10 @@
   position: absolute;
   z-index: 10;
   overflow: auto;
-  height: 556px;
-  padding: 54px 140px 54px 160px;
+  height: 100%;
+  padding: 54px 160px 54px 180px;

```

Comment:
semantically, I think you mean "margin" here, unless you're talking about _inside_ the modal.